base/linalg/lapack.jl

```diff
@@ -5040,10 +5040,10 @@ for (syev, syevr, sygvd, elty, relty) in
             lwork = BlasInt(-1)
             iwork = Vector{BlasInt}(1)
             liwork = BlasInt(-1)
-            rwork = Array{$relty,0}()
+            rwork = Vector{$relty}(1)
```

Comment:
Why change from zero-dimensional `Array` to a single-element one-dimensional `Array`?